the-shore/patterns/header-homepage.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Title: header-homepage
+ * Slug: the-shore/header-homepage
+ * Categories: hidden
+ * Inserter: no
+ */
+?>
+<!-- wp:cover {"url":"<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/hotel_cover_image.jpg","dimRatio":20,"overlayColor":"primary","isUserOverlayColor":true,"minHeight":672,"minHeightUnit":"px","style":{"spacing":{"padding":{"right":"0","left":"0","top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-cover" style="padding-top:0;padding-right:0;padding-bottom:0;padding-left:0;min-height:672px"><span aria-hidden="true" class="wp-block-cover__background has-primary-background-color has-background-dim-20 has-background-dim"></span><img class="wp-block-cover__image-background" alt="<?php echo __('', 'the-shore');?>" src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/hotel_cover_image.jpg" data-object-fit="cover"/><div class="wp-block-cover__inner-container"><!-- wp:group {"align":"full","style":{"dimensions":{"minHeight":"100vh"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"stretch"}} -->
```

Comment:
It's not an accessible preview of the theme, so it should be a blocker. We design the content for sighted users to get an idea of the theme even though it will be replaced, so we should the same for everyone. People look to these themes as an example of how to do things, so we should do it correctly. It should be straightforward to hardcode some alt text into these images.

> CBT that adds-in placeholder alt text in on export

We'll need accurate alt text, so hardcoding it is better.

Reply:
My workflow for these updates is I make the adjustments directly in the editor, save changes to the theme with CBT and export changes as a PR.

Reply:
But the alt text can be changed in the editor, doesn't need to be done in the code